solutions/gold/usaco-862.mdx

```diff
@@ -0,0 +1,100 @@
+---
+id: usaco-862
+source: USACO Gold 2018 December
+title: Cowpatibility
+author: Ryan Chou
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_cowpatibility_gold_dec18.html)
+
+<Spoiler title="Hint">
+Instead of trying to find the number of incompatible pairs, try to find the number of compatible pairs!
+
+This is known as [complementary counting](https://artofproblemsolving.com/wiki/index.php/Complementary_counting) :D
```

Comment:
maybe remove the emoticon? i, personally, don't have any issue w/ it but it might not match the guide style

Reply:
oops my bad